src/Containers/FileUpload/Container.ts

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import {flex} from '@Utils/Mixins'
+
+export interface IContainerProps{
```

Comment:
Why not extend Div?

Reply:
I think it's not necessary in the case of styled components.

Reply:
But then why are you redeclaring are the styled props?

If you want all the style props exposed without doing this manually and having them optional why not just do

Partial<Pick<Div, Style>>

Reply:
Sorry, I don't get it. What is Style? I have personal options (properties) defined by me which I need. That's why I declare them.

Reply:
I do not understand what is happening here at all. Can you try to explain what EXACTLY you are trying to do here.
DragEnter comes from https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.dragevent.html 
DragEvent, i don't know why you are creating custom types? Margin is already from div elements etc

Reply:
Is not DrageEnter, is isDragEnter, which is a boolean. and margin is not a valid div attribute, as far as I know